CHANGELOG.md

```diff
@@ -4,12 +4,13 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
-## [Unreleased]
+## [0.12.0] - 2022-08-27
```

Comment:
@plannigan if you're comfortable with me releasing these 3.10 changes, I'll also try running the version bump process. Meanwhile, could you review the rest of the work?

Reply:
Can you please remove this line from the PR? The [repository has a process](https://wayfair-incubator.github.io/columbo/0.11.0/development-guide/#publishing-a-new-version) for updating the version number, which is incorporated in the the release process. Also, this is not the only place that the version number appears and needs to be incremented.

I'd also like to get #357 addressed in this release as well.

Reply:
Ok. I'll also update CONTRIBUTING.md to make that point more clear.